src/metabase/driver/druid/query_processor.clj

```diff
@@ -443,15 +443,15 @@
                                                      {:dimension (->rvalue field)
                                                       :direction direction}))))
 
-(defmethod handle-order-by ::grouped-timeseries [_ {{ag-type :aggregation-type} :aggregation, [breakout-field] :breakout, [{field :field, direction :direction}] :order-by} druid-query]
-  (let [field             (->rvalue field)
-        breakout-field    (->rvalue breakout-field)
-        sort-by-breakout? (= field breakout-field)]
-    (if (and sort-by-breakout?
-             (= direction :descending))
-      (assoc druid-query :descending true)
-      druid-query)))
+(defn sort-by-timestamp [field direction druid-query]
+  (assoc druid-query :descending (and (= (->rvalue field) "timestamp")
```

Comment:
@moumny if I understand correctly a Druid timestamp column isn't always called `timestamp`, right? What happens in those cases? This sounds like it would only work if they named the column a certain way